src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -2,39 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.ExceptionServices;
 using Xunit;
-using Xunit.NetCore.Extensions;
 
 namespace System.Tests
 {
     // No appdomain in UWP or CoreRT
     [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18718")]
```

Comment:
We should have an issue tracking removing skipping on uapaot. This falls into the "scary" category ie there is no issue tracking it just note in https://github.com/dotnet/corefx/pull/19331.